### DISCUSSION TITLE
[Idea] Bootable CD Installer for the memory card image.

### DISCUSSION THREAD

**Discussion by JoseAaronLopezGarcia** [brad-lin/FreePSXBoot#66] [Ideas]
I noticed that all installation methods require flashing an image into the memory card using some form of specialized hardware, either using a modded PS2 or some PC adapter.
Why not use the PS1 itself?
It should be possible to make a PS1 bootable CD to flash the image using the PS1 itself.
You can use the swap method to boot the installer CD the first time.

**Comment by brad-lin** [comment]
The Unirom boot disc can do just that :)

**Reply by jeffbrl** [reply]
It seems to me like Unirom would need to include FreePSXBoot images if you want to avoid using a PS2 or specialized hardware to write the image to a memory card. I acknowledge that I may be missing something though. :-)

**Reply by brad-lin** [reply]
If you download the Unirom boot disc from here: https://github.com/JonathanDotCel/unirom8_bootdisc_and_firmware_for_ps1/releases/tag/8.0.J

Then burn it and run it, you can install FreePSXBoot directly from a PlayStation.

It is not possible to do that when running FreePSXBoot from a memory card, because there is simply not enough space to store the images for all BIOSes on a memory card.

I hope this answers your question :)

**Reply by jeffbrl** [reply]
@brad-lin Thank you. I appreciate your answering questions from someone new to the PS1 hacking scene.
